src/store/redusers/get_data_reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { SET_DATA } from '../constants';
+
+const initialState = {
+  inverter_one: [],
```

Comment:
This structure doesn't seem to be extendable. What if I will install the third inverter? Please remember, the app should be extensible and flexible enough. If you don't have better ideas about data structure you may just re-use the data structure from the api

Reply:
Done